config/200-controller-clusterrole.yaml

```diff
@@ -0,0 +1,27 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: knative-eventing-controller
+rules:
+  - apiGroups: [""]
+    resources: ["namespaces", "secrets", "configmaps", "services", "events", "serviceaccounts"]
+    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources: ["customresourcedefinitions"]
+    verbs: ["get", "list", "create", "update", "delete", "patch", "watch"]
```

Comment:
Does the controller really need write permissions on CRDs?

Reply:
Do these permissions apply to CRDs or all the custom resources whose type is a CRD? If the former, why do we need any permissions on CRDs?

Reply:
left over cruft, removed.